core/cas-server-core-webflow/src/main/java/org/apereo/cas/web/flow/config/CasWebflowContextConfiguration.java

```diff
@@ -185,7 +185,11 @@ public LocaleChangeInterceptor localeChangeInterceptor() {
     @Bean
     @ConditionalOnMissingBean(name = "resourceUrlProviderExposingInterceptor")
     public ResourceUrlProviderExposingInterceptor resourceUrlProviderExposingInterceptor() {
-        return new ResourceUrlProviderExposingInterceptor(resourceUrlProvider.getObject());
+        val provider = resourceUrlProvider.getIfAvailable();
+        if (provider != null) {
+            return new ResourceUrlProviderExposingInterceptor(provider);
+        }
+        return null;
```

Comment:
What does it mean for a bean to return a null reference? Wouldn't this cause a crash?

Reply:
No, it's fine, thank you. I think this might internally translate to a NullBean instance.